test/integration_test.rb

```diff
@@ -285,6 +285,67 @@ def test_incremental_commit_does_not_automatically_add_files
     end
   end
 
+  def test_commit_with_message_type_message
+    Dir.chdir(TMP_DIR) do
+      create_git_repo
+
+      in_git_repo do
+        RakeCommit::Shell.system "touch new_file"
+        RakeCommit::Shell.backtick("printf 'test-author\ntest-feature\ntest-message\n' | ../../../bin/rake_commit -m message", false).split("\n")
+
+        log_lines = RakeCommit::Shell.backtick("git log").split("\n")
+        assert_operator log_lines.grep(/test-feature - test-message/).length, :>, 0
+      end
+    end
+  end
+
+  # Ruby versions prior to 2.0 did not support pre_input_hook. Behavior is different if
```

Comment:
I'm just going to drop support for pre-2.0 versions. It's EOL'ed anyway: https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/

So you can delete this stuff.